front-end/src/components/molecules/side-bar/side-bar.tsx

```diff
@@ -1,17 +1,53 @@
+import { CircleUser, LayoutGrid, LogOut, Plus } from "lucide-react";
+import Link from "next/link";
+
 export default function Sidebar() {
+  const dashboardElements = [
```

Comment:
also, move the `dashboardElements` out of the component.
Since there's no state in it, and it's a constant, remove it and leave it out of the function itself  😇